lib/takeaway.rb

```diff
@@ -0,0 +1,27 @@
+class Takeaway
+
+  attr_reader :items, :your_order, :total
+
+  def initialize
+    @items = {"pizza" => 10, "chips" => 2, "wings" => 5, "coke" => 1, "chicken" => 6, "pasta" => 7.50}
```

Comment:
This (@items) looks like a good candidate for something to extract out into a separate class. It perhaps isn't the direct responsibility of a restaurant *interface* to know the details of what's on the menu - it can look it up.